test/metabase/query_processor/util/nest_query_test.clj

```diff
@@ -0,0 +1,694 @@
+                                                                     ::add/source-alias  "LATITUDE"
+                                                                     ::add/desired-alias "LATITUDE"
+                                                                     ::add/position      3}]
+                                               [:field %longitude   {::add/source-table  $$venues
+                                                                     ::add/source-alias  "LONGITUDE"
+                                                                     ::add/desired-alias "LONGITUDE"
+                                                                     ::add/position      4}]
+                                               [:field %price       {::add/source-table  $$venues
+                                                                     ::add/source-alias  "PRICE"
+                                                                     ::add/desired-alias "PRICE"
+                                                                     ::add/position      5}]
+                                               [:expression "double_price" {::add/desired-alias "double_price"
+                                                                            ::add/position      6}]]}
+                 :breakout     [[:field %price {::add/source-table  ::add/source
+                                                ::add/source-alias  "PRICE"
+                                                ::add/desired-alias "PRICE"
+                                                ::add/position      0}]]
+                 :aggregation  [[:aggregation-options [:count] {:name               "count"
+                                                                ::add/desired-alias "count"
+                                                                ::add/position      1}]]
+                 :fields       [[:field "double_price" {:base-type          :type/Float
+                                                        ::add/source-table  ::add/source
+                                                        ::add/source-alias  "double_price"
+                                                        ::add/desired-alias "double_price"
+                                                        ::add/position      2}]]
+                 :order-by     [[:asc [:field %price {::add/source-table  ::add/source
+                                                      ::add/source-alias  "PRICE"
+                                                      ::add/desired-alias "PRICE"
+                                                      ::add/position      0}]]]})
+              (nest-expressions
+               (mt/mbql-query venues
+                 {:expressions {:double_price [:* $price 2]}
+                  :breakout    [$price]
+                  :aggregation [[:count]]
+                  :fields      [[:expression "double_price"]]})))))
+
+(deftest nest-expressions-test-2
```

Comment:
Add testing context string or comment explaining what this one is for (as opposed to above)?

Reply:
fixed